src/components/Dashboard/TodoList.vue

```diff
@@ -408,6 +436,7 @@ export default {
       confirmEnabled: false,
       currentFilingId: null,
       panel: null as number, // currently expanded panel
+      payAPiUrl: sessionStorage.getItem('PAY_API_URL'),
```

Comment:
I'd rather see this as a getter method, as we do elsewhere.

I believe by declaring this here, it's assigned when the component is created, so the session item needs to exist before that. Instead of making that assumption, by making this a getter, it becomes reactive, which theoretically means it's updated where its content changes, so it's always up to date.

Reply:
done

Reply:
Fixed.